assets/src/components/list-item/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export function ListItem( { className = '', heading, items } ) {
+	return (
+		<ul className={ classnames( 'list-items', className ) }>
+			{
+				( () => {
+					if ( heading ) {
+						return (
+							<li className="list-items__item">
+								<h4 className="list-items__heading">
+									{ heading }
+								</h4>
+							</li>
+						);
+					}
+
+					return null;
+				} )()
+			}
```

Comment:
How about simplifying it to something like this:

```suggestion
			{ heading && (
				<li className="list-items__item">
					<h4 className="list-items__heading">
						{ heading }
					</h4>
				</li>
			) }
```